conftest.py

```diff
@@ -8,6 +8,13 @@
 from django.db import connections
 from django.test.testcases import TransactionTestCase
 
+from saleor.tests.utils import prepare_test_db_connections
```

Comment:
Should we user relative imports here?

Reply:
`conftest.py` is in the root catalog, next to `saleor` module, relative import cannot be used here.